website/docs/dependencies/0070-install-snap-packages.md

```diff
@@ -0,0 +1,46 @@
+# Install Snap Packages
+Install packages available using the Snap Package Manager.
+
+- [Official Snap Documentation](https://snapcraft.io/docs/snap-tutorials)
+
+:::info
+Installing snap packages depends on the successful installation of the snapd service. Follow the example below to make sure the snapd service is installed correctly using the [LinuxPackageInstallation dependency](./0010-install-linux-packages.md) before adding this one.
+:::
+
+## Supported Platform/Architectures
+* linux-x64
+* linux-arm64
+
+## Profile Component Parameters
+The following section describes the parameters used by the individual component in the profile.
+
+| **Parameter** | **Required** | **Description**                                                                                                 |
+|---------------|--------------|-----------------------------------------------------------------------------------------------------------------|
+| Packages      | Yes          | Comma delimitered list of packages to be installed via snap.                                              |
```

Comment:
There is another parameter for allow upgrade. Either remove that support, or include it here as parameters